### DISCUSSION TITLE
Why does the highres.fix downsample even with dimensions divisible by 64?

### DISCUSSION THREAD

**Discussion by Taikakim** [TheLastBen/fast-stable-diffusion#1336] [General]
For an example, 2,5x scaling on a 512x640 image would be 1280x1600. Still during the process the init image gets downsampled to 320x400. Why is this? This is using the LDSR upscaler.

**Comment by TheLastBen** [comment]
I think you will get better answers for that issue in https://github.com/AUTOMATIC1111/stable-diffusion-webui
